src/controllers/AuthController.js

```diff
@@ -31,6 +34,53 @@ const forgotPassword = async (request, response) => {
   }
 };
 
-export default {
-  forgotPassword
+/**
+ * Update user verified status
+ *
+ * @param {object} req
+ * @param {object} res
+ * @returns {json} - json
+ */
+
+const updateStatus = async (req, res) => {
+  const user = await User.findOne({ where: { id: req.params.id } });
+  const secret = process.env.ACCOUNT_VERIFICATION_SECRET;
+  const token = await jwt.verify(req.params.token, secret, (err, decoded) => {
```

Comment:
if you see yourself doing a lot of chaining, you can destructure. In this case, you can do `const { params: { token } } = req` and use `token` directly instead of `req.params.token`

Reply:
done